src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -593,6 +600,7 @@ namespace HVACVariableRefrigerantFlow {
             CondInletHumRat = PsyWFnTdbTwbPb(CondInletTemp, OutdoorWetBulb, OutdoorPressure);
         } else if (VRF(VRFCond).CondenserType == DataHVACGlobals::WaterCooled) {
             CondInletTemp = OutdoorDryBulb; // node inlet temp from above
+            OutdoorWetBulb = CondInletTemp; // for watercooled
```

Comment:
Yes, this value is used somewhere down a few lines codes. The performance curve is evaluated depending on the HeatingPerformanceOATType curve but not on the condenser type. see this code below:
```         {
                auto const SELECT_CASE_var(VRF(VRFCond).HeatingPerformanceOATType);
                if (SELECT_CASE_var == DataHVACGlobals::DryBulbIndicator) {
                    TotHeatCapTempModFac = CurveValue(VRF(VRFCond).HeatCapFT, InletAirDryBulbC, CondInletTemp);
                    TotHeatEIRTempModFac = CurveValue(VRF(VRFCond).HeatEIRFT, InletAirDryBulbC, CondInletTemp);
                } else if (SELECT_CASE_var == DataHVACGlobals::WetBulbIndicator) {
                    TotHeatCapTempModFac = CurveValue(VRF(VRFCond).HeatCapFT, InletAirDryBulbC, OutdoorWetBulb);
                    TotHeatEIRTempModFac = CurveValue(VRF(VRFCond).HeatEIRFT, InletAirDryBulbC, OutdoorWetBulb);
                } else {
                    TotHeatCapTempModFac = 1.0;
                    TotHeatEIRTempModFac = 1.0;
                }
            } ```

Reply:
I think that was the whole point of this issue, whenf the user entered WetBulbIndicator then the user noticed the outdoor WB was used instead of the inlet water temp. So either we didn't allow WetBulbIndicator with water cooled or we fixed it to give the correct results regardless.